image_geometry/CMakeLists.txt

```diff
@@ -1,38 +1,43 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(image_geometry)
 
-find_package(catkin REQUIRED sensor_msgs)
-find_package(OpenCV REQUIRED)
+find_package(ament_cmake REQUIRED)
 
-catkin_package(CATKIN_DEPENDS sensor_msgs
-               DEPENDS OpenCV
-               INCLUDE_DIRS include
-               LIBRARIES ${PROJECT_NAME}
-)
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra")
+endif()
 
-catkin_python_setup()
+find_package(OpenCV REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(Boost REQUIRED)
```

Comment:
@clalancette If you got rid of the boost calls, the find_package and the dependency in the package.xml should be removed. Keeping those causes the jobs running without boost to fail: http://ci.ros2.org/job/ci_linux/2549/consoleFull